### PR TITLE
Rewrite the methods section of the IP class

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by cbscribe** [comment]
Looks great - thanks for your help!

File: doc/base/classes.xml [hunk 1]
```diff
@@ -22114,6 +22114,7 @@
 			<argument index="0" name="hostname" type="String" default="&quot;&quot;">
 			</argument>
 			<description>
+                Erases all cached references to a given hostname. If no hostname is given then all stored IP addresses are deleted from the cache.
```
Comment (djrm) [inline doc/base/classes.xml]: indentation issue.

File: doc/base/classes.xml [hunk 2]
```diff
@@ -22122,13 +22123,14 @@
...
 			<description>
-				Erase a queue ID, removing it from the queue if needed. This should be used after a queue is completed to free it and enable more queries to happen.
+				Erases a queue ID, and removes it from the queue if needed. This should be used to free a queue after it has completed to enable more queries to happen.
 			</description>
 		</method>
 		<method name="get_local_addresses" qualifiers="const">
 			<return type="Array">
 			</return>
 			<description>
+                Returns an array that contains all current IPv4 and IPv6 addresses of the user's device.
```
Comment (djrm) [inline doc/base/classes.xml]: same here

Comment (deliciousfudge) [reply doc/base/classes.xml]: Thanks, will take a look at those now

**Comment by akien-mga** [comment]
Conflicts with #11140?

**Comment by deliciousfudge** [comment]
Ah yes, sorry. Just noticed that the corrections in #11140 have made everything in the commit redundant. Will close it off.

**Pull request closed without merging**